### ISSUE TITLE
Det bør vare mulig å customisere Fødselsnummer validasjon.

### ISSUE THREAD

**Issue by alpet** [bekkopen/NoCommons#56]
Spesielt når det er syntetiske genrerte fødselsnummer man trenger å ha mer relaxed tilnarming til fødselsnummer validering.
Det er best hvis man har muilghet å gjøre override på et eller annet aspekt av validasjonen. Men mulighet for å slå det AV vil også vare en rask måtte å gå rundt dette.

**Comment by eivinhb** [comment]
Kan du forklare mer om behovet? Hvorfor ønsker du i noen tilfeller å ikke utføre valideringen?

**Comment by janhoy** [comment]
Er ikke det støttet allerede i https://github.com/bekkopen/NoCommons/blob/master/src/main/java/no/bekk/bekkopen/person/FodselsnummerValidator.java#L40-L45 ?

**Comment by eivinhb** [comment]
Ja, syntetiske nummer er støttet. I hvert fall de skikkelige syntetiske nr. vi kommer ikke til å lage en innebygget feature toggle for å slå av validering.
